beetsplug/beatport4.py

```diff
@@ -138,9 +138,9 @@ def __init__(self, data):
                 self.length = timedelta(minutes=int(min), seconds=int(sec))
             except ValueError:
                 pass
-        if 'key' in data and data['key']['name']:
+        if data.get('key') and data['key']['name']:
             self.initial_key = self._normalize_key(str(data['key']['name']))
-        if 'bpm' in data and data['bpm']:
+        if data.get('bpm') and data['bpm']:

```

Comment:
Is the further check anyhow useful now?
```suggestion
        if data.get('bpm'):

```

Reply:
Yes, haha certainly, that's better ;-) I was not aware of githubs "fix suggestion feature". nice!